### PR TITLE
chore: prepare1.0.23 dev and SNAPSHOT

### LINKED ISSUE #7655: fix: chore: prepare1.0.23 dev and SNAPSHOT -autocreated
This issue was created because no open issues were referenced in your PR (#7654).

### PR DESCRIPTION
**Pull request by moabu**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #issue-number-here

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Closes #7655,

### CODE REVIEW

**Comment by mo-auto** [comment]
Error: Hi @moabu, You did not reference an open issue in your [PR](7654). I attempted to create an [issue](#7655) for you.    
 Please update that [issues'](#7655) title and body and make sure I correctly referenced it in the above PRs body.

**Review by pujavs — DISMISSED** [review]

**Review by yuriyz — APPROVED** [review]

File: charts/janssen/Chart.yaml [hunk 1]
```diff
@@ -44,52 +42,52 @@ maintainers:
...
 
     - name: link
       condition: global.link.enabled
-      version: 1.0.22
+      version: 1.0.23-dev
 
     - name: saml
       condition: global.saml.enabled
-      version: 1.0.22
+      version: 1.0.23-dev
```
Comment (iromli) [inline charts/janssen/Chart.yaml:93]: The following sub-chart should be added:

```
- name: cn-istio-ingress
  condition: global.istio.ingress
  version: 1.0.23-dev
```

File: charts/janssen-all-in-one/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # janssen-all-in-one
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen-all-in-one/README.md:4]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/auth-server-key-rotation/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # auth-server-key-rotation
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/auth-server-key-rotation/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/auth-server/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # auth-server
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/auth-server/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/casa/README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # casa
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/casa/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/cn-istio-ingress/README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 # cn-istio-ingress
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/cn-istio-ingress/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/config-api/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 # config-api
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/config-api/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/config/README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # config
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/config/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/fido2/README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # fido2
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/fido2/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/link/README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # link
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/link/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/nginx-ingress/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # nginx-ingress
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/nginx-ingress/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/opendj/README.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # opendj
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/opendj/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/persistence/README.md [hunk 13]
```diff
@@ -1,6 +1,6 @@
 # persistence
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/persistence/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/saml/README.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 # saml
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/saml/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: charts/janssen/charts/scim/README.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 # scim
 
-![Version: 1.0.22](https://img.shields.io/badge/Version-1.0.22-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.22](https://img.shields.io/badge/AppVersion-1.0.22-informational?style=flat-square)
+![Version: 1.0.23-dev](https://img.shields.io/badge/Version-1.0.23--dev-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.23-dev](https://img.shields.io/badge/AppVersion-1.0.23--dev-informational?style=flat-square)
```
Comment (iromli) [inline charts/janssen/charts/scim/README.md:3]: The extra `-` need to be removed:

```
1.0.23--dev
```

File: docker-jans-all-in-one/Dockerfile [hunk 16]
```diff
@@ -3,7 +3,7 @@
 # ==============
 
 # original Janssen base version (without suffix)
-ARG BASE_VERSION=1.0.22
+ARG BASE_VERSION=1.0.23-SNAPSHOT
```
Comment (iromli) [inline docker-jans-all-in-one/Dockerfile]: The `-SNAPSHOT` suffix should be removed.

**Review by iromli — CHANGES_REQUESTED** [review]

**Review by yurem — APPROVED** [review]

**Review by jgomer2001 — DISMISSED** [review]

**Review by uprightech — APPROVED** [review]

**Review by iromli — DISMISSED** [review]

**Pull request merged**